src/modules/common/components/layout.jsx

```diff
@@ -2,18 +2,21 @@ import React from 'react';
 import { connect } from 'react-redux';
 import SiteNav from './site-nav';
 import SiteFooter from './site-footer';
+import Landing from './landing';
 
 const Layout = ({ user, loginInitialized, children }) =>
-  <div>
-    <header>
+  <div className="layout">
+    <header className="layout__header">
```

Comment:
Is `layout__header` gonna be defined in the styles later? Also, similar comments about unnecessary nesting...